src/components/campaign_poster/CampaignPoster.module.scss

```
@@ -2,35 +2,57 @@
 
 .campaignPoster {
   display: flex;
-  flex-direction: column;
-  justify-content: flex-end;
-  box-shadow: $campaignBoxShadow;
-  height: 500px;
-  max-height: 500px;
-  max-width: 500px;
-  width: inherit;
-  margin: auto;
-  border-radius: 8px;
-  color: $white;
-  padding: 18px;
-  box-sizing: border-box;
-  background-blend-mode: overlay;
-  background-color: $activeOverlay;
-  background-position: center;
-  & h3 {
-    margin-block-end: 0.4em;
+  justify-content: center;
+  margin-bottom: 2rem;
+  box-shadow: 0 0 10px 0px rgba($color: #000000, $alpha: 0.08);
+  @media (max-width: $medium) {
+    flex-wrap: wrap;
+    box-shadow: none;
   }
-  & h4 {
-    line-height: 1;
+  transition: 0.5s;
+  &:hover {
+    transform: scale(1.01);
+  }
+  .left {
+    width: 50%;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    overflow: hidden; // edge case placeholder
+    @media (max-width: $medium) {
+      max-height: 200px;
+      height: auto;
+      width: 100%;
+    }
+    img {
+      width: 100%;
+      height: auto;
+    }
```

Not sure if you want to go this direction, but some images fit a little better out of our defined size variables.
`img {`
`   width: 100%;`
`   height: 100%;`
`   object-fit: cover`
`}`

---

Yep thats a good shout! Btw @tunangst you can give suggestions with one of the buttons (The one next to header text) and that means they can just be committed directly. 

eg
```suggestion
    img {
      width: 100%;
      height: auto;
      object-fit: cover;
    }
```

---

Yes great idea - thanks @tunangst !